lib/oelite/baker.py

```diff
@@ -76,9 +76,12 @@ def add_bake_parser_options(parser):
 
 
 def add_show_parser_options(parser):
+    import optparse
     parser.add_option("--nohash",
                       action="store_true",
                       help="don't show variables that will be ignored when computing data hash")
+    parser.add_option("--nodynvar",
+                      action="store_true", help=optparse.SUPPRESS_HELP)
```

Comment:
No, this doesn't expand less. It prevents "unexpanding", so in that sense it expands more (e.g., showing DATETIME="20160102030405" instead of DATETIME="${DATETIME}"). And yes, I've occasionally been confused by the not fully expanded stuff, but now I understand why it happens, so I'm not going to push very hard for this option to get in if others find it entirely useless.